README.md

```diff
@@ -25,88 +36,51 @@ account=facu
 default_vault=awesome-vault
 ```
 
-## CLI Sections
-
-The command containse two sections:
-
-* `search` : For seeking for a stored item
-* `item` : For adding and getting an specific item
-
-```profile
-usage: cabinet (sub-commands ...) [options ...] {arguments ...}
+## CLI Overview
 
-Cabinet's cli client for managing vaults.
+This are the now implemented features on the cli, hopefully this is self explanatory.
 
-commands:
-
-  item
-    Item Controller
-
-  search
-    Search Controller
-
-optional arguments:
-  -h, --help     show this help message and exit
-  --debug        toggle debug output
-  --quiet        suppress all output
-  -v, --version  show program's version number and exit
 ```
+➜ cab
+Usage: cab [OPTIONS] COMMAND [ARGS]...
 
-### Searching
-
-```profile
-usage: cabinet (sub-commands ...) [options ...] {arguments ...}
-
-Search Controller
+Options:
+  -a, --account TEXT  Specify an account to use
+  -v, --vault TEXT    Specify a vault to use
+  --help              Show this message and exit.
 
-commands:
-
-  default
-    Get all the items in the vault.
-
-positional arguments:
-  extra_arguments
-
-optional arguments:
-  -h, --help            show this help message and exit
-  --debug               toggle debug output
-  --quiet               suppress all output
-  -a ACCOUNT, --account ACCOUNT
-                        Use this account to authenticate
-  -v VAULT, --vault VAULT
-                        Use this vault
-  -s, --show-tags       Show items with tags.
-  -t TAG, --tag TAG     Filter by tag
-  --tags TAGS           Filter by multiple tags
-```
+Commands:
+  add     Add an item to cabinet
+  get     Get an item from cabinet
+  rm      [not implemented] Remove an item from cabinet
+  search  Item searching on cabinet
 
-### Adding/Getting
+➜ cab add --help
+Usage: cab add [OPTIONS] NAME
 
-```profile
-usage: cabinet (sub-commands ...) [options ...] {arguments ...}
+  Add an item to cabinet
 
-Item Controller
+Options:
+  -t, --tag TEXT      Specify tags for the item
+  -c, --content TEXT  The item content
+  -e, --edit-content  The item content
+  --help              Show this message and exit.
 
-commands:
+➜ cab get --help
+Usage: cab get [OPTIONS] NAME
 
-  add
-    Add an item to the vault.
+  Get an item from cabinet
 
-  get
-    Get an item from the vault.
+Options:
+  --help  Show this message and exit.
+ivan in [~/data/Devel/Cabinet/cabinet-cli]
+(venv:cabinet) (git:migrate-to-click ✘) ➜ cab search --help
```

Comment:
Oh, this wasn't intentional, I removed the other ones when I copy/pasted from the terminal.
Removed.